services/web/client/source/class/qxapp/About.js

```diff
@@ -0,0 +1,73 @@
+/* eslint no-warning-comments: "off" */
+qx.Class.define("qxapp.About", {
+  extend: qx.ui.window.Window,
+  type: "singleton",
+
+  construct: function() {
+    this.base(arguments, this.tr("About"));
+    this.set({
+      layout: new qx.ui.layout.VBox(),
+      contentPadding: 20,
+      showMaximize: false,
+      showMinimize: false,
+      resizable: false,
+      centerOnAppear: true
+    });
+    this.__populateEntries();
+  },
+
+  members: {
+    __populateEntries: function() {
+      // All these items and versions should be red from a file
+      this.add(this.__createEntry("oSPARC UI", "3.38", "https://github.com/ITISFoundation/osparc-simcore"));
```

Comment:
You should also pin the commit id for the client, this way we reproduce exactly with the code that produced that UI

To get the commit id
```
git log --pretty=tformat:"%h" -n1 services/web/client/
```
and if the code was not versioned (i.e. dirty), you could detect it as well
```
git diff --quiet services/web/client || echo 'dirty'
```

Reply:
I think with 

```
qx compile --set osparc.commit='"'`git diff --quiet . && git log --pretty=tformat:"%h" -n1 || echo dirty`'"'
```
you could get this into the qooxdoo env like the compiler version and other info

Reply:
Great Idea Tobi!!